application/src/test/resources/speedtest/travelSearch-expected-results-sr.csv

```diff
@@ -1,5 +1,5 @@
 tcId,nTransfers,duration,cost,walkDistance,startTime,endTime,agencies,modes,routes,stops,details
-1,0,56m36s,0,1458,12:03:49,13:00:25,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:11:20 12:49:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m6s
-1,2,1h4m19s,0,2813,12:10:47,13:15:06,TriMet,BUS,73|70|9,prt:7106|prt:4056|prt:4055|prt:4536|prt:4538|prt:143,Walk 2m19s ~ NE 21st & Clackamas(7106) ~ BUS 73 12:13:06 12:16:45 ~ NE Multnomah & 9th(4056) ~ Walk 35s ~ NE Multnomah & 9th(4055) ~ BUS 70 12:19:38 12:35 ~ SE Powell & Milwaukie(4536) ~ Walk 1m18s ~ SE Powell & Milwaukie(4538) ~ BUS 9 12:39 12:42:50 ~ SW Arthur & 1st(143) ~ Walk 32m16s
-1,0,56m36s,0,1458,12:18:49,13:15:25,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:26:20 13:04:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m6s
-2,0,57m54s,0,1507,12:02:31,13:00:25,TriMet,BUS,8,prt:6779|prt:6548,Walk 8m14s ~ NE 15th & Weidler(6779) ~ BUS 8 12:10:45 12:49:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m6s
+1,0,56m33s,0,1458,12:03:49,13:00:22,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:11:20 12:49:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m3s
+1,2,1h4m16s,0,2813,12:10:47,13:15:03,TriMet,BUS,73|70|9,prt:7106|prt:4056|prt:4055|prt:4536|prt:4538|prt:143,Walk 2m19s ~ NE 21st & Clackamas(7106) ~ BUS 73 12:13:06 12:16:45 ~ NE Multnomah & 9th(4056) ~ Walk 35s ~ NE Multnomah & 9th(4055) ~ BUS 70 12:19:38 12:35 ~ SE Powell & Milwaukie(4536) ~ Walk 1m18s ~ SE Powell & Milwaukie(4538) ~ BUS 9 12:39 12:42:50 ~ SW Arthur & 1st(143) ~ Walk 32m13s
+1,0,56m33s,0,1458,12:18:49,13:15:22,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:26:20 13:04:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m3s
+2,0,57m51s,0,1507,12:02:31,13:00:22,TriMet,BUS,8,prt:6779|prt:6548,Walk 8m14s ~ NE 15th & Weidler(6779) ~ BUS 8 12:10:45 12:49:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m3s
```

Comment:
Do you know why all of these speed tests are 3 seconds quicker? Ideally changing the internal angle representation should not change travel time.